src/main/webapp/app/exam/participate/exam-navigation-sidebar/exam-navigation-sidebar.component.html

```diff
@@ -70,7 +70,7 @@
         id="test-collapse"
         class="auto-collapse double-arrow mb-2"
         [ngClass]="{ 'menu-closed': isCollapsed }"
-        [ngbTooltip]="(isCollapsed ? 'Expand' : 'Collapse') + ' Menu (Ctrl + M)'"
```

Comment:
We should stick with `Menu` or something like `Navigation` or `Navigationbar` (even though it's a bit long), otherwise we'll have `Sidebar` twice 🙃